### PR TITLE
[16.0][MIG] mass_mailing_resend: Migration to 16.0

### PR DESCRIPTION
**Pull request by nguyenhk99**

### CODE REVIEW

**Review by StephaneMangin — APPROVED** [review]

**Review by leemannd — APPROVED** [review]

File: mass_mailing_resend/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -2,3 +2,5 @@
 
   * Pedro M. Baeza
   * Ernesto Tejeda
+* `Trobz <https://trobz.com>`_:
+    * nguyenhk <nguyenhk@trobz.com>
```
Comment (pedrobaeza) [inline mass_mailing_resend/readme/CONTRIBUTORS.rst]: Incorrect RST syntax. Please leave an empty line.

Comment (nguyenhk99) [reply mass_mailing_resend/readme/CONTRIBUTORS.rst]: PR has been updated. Thank you.

Comment (pedrobaeza) [reply mass_mailing_resend/readme/CONTRIBUTORS.rst]: It still seems incorrect, as the indentation is higher than the required.

**Pull request merged**